### ISSUE TITLE
Storage Cells Merging

### ISSUE THREAD

**Issue by unejamzezak** [62832/MEGACells#67]
Not really sure if this is intended to work this way, I made multiple 16M storage cells and when I transferred the items from pre-existing storage cells using the ME IO Port from AE2, all the storage cells ended up merging somehow.

https://imgur.com/qTa9OwU

EDIT -- When putting the storage cells into the ME IO Port, it ends up pulling out 0 items, which makes me believe it could be a visual bug? When I start transferring in items from the network into the storage cell itself it starts to fill up, and when I pull the items out again it still holds the same value.
Restarting the client does not help, I am playing on ATM 9, the mod version is : megacells-forge-2.1.0-beta.4-1.20.1

https://imgur.com/89paBxN

https://imgur.com/eWN7gzU

**Comment by unejamzezak** [comment]
Not entirely sure if this is a bug or not, could have been that my storage was just full since I have miscellaneous items that are one of, so it could have been rendering the storage cells as full and that's why items weren't being moved. I decided to just make multiple ME drives and use more storage cells

**Comment by 62832** [comment]
Has that worked for you, in that case? I have to say this is a rather confusing issue and report and I can't say I'm really sure what might cause it, given that M-tier cells are just instances of AE2's own cells.

**Comment by unejamzezak** [comment]
Yeah I made multiple ME Drives and just made a ton of 1M storage cells just incase, ever since creating them they haven't filled up even once and it's been using my main drives which I showed in the video. I'm avoiding using the ME IO Port just incase, which before I had the 16M item cells they weren't being used, once I moved all items from the 256k storage cells from AE2 into the 16M cells they all filled up at the same time which was rather strange, but I have in total 30 ME Drives with 300 storage cells as to avoid the issue.
